packages/fondue/src/components/FrontifyPattern/FrontifyPattern.stories.tsx

```diff
@@ -7,7 +7,7 @@ import { PatternDesign, PatternScale, PatternTheme } from '@foundation/Pattern';
 import { FrontifyPattern, type FrontifyPatternProps } from './FrontifyPattern';
 
 export default {
-    title: 'Components/FrontifyPattern',
+    title: 'Legacy Components/FrontifyPattern',
```

Comment:
Not used outside frontify, can be deleted. The pattern never have been updated anyway and is not on brand anymore